package.nls.json

```diff
@@ -193,6 +193,9 @@
     "AWS.message.info.stepFunctions.publishStateMachine.creating": "Creating state machine '{0}' in {1}...",
     "AWS.message.info.stepFunctions.publishStateMachine.updateSuccess": "Successfully updated state machine '{0}'",
     "AWS.message.info.stepFunctions.publishStateMachine.updating": "Updating state machine '{0}' in {1}...",
+    "AWS.message.info.yaml.prompt": "You can access additional AWS SAM functionality by installing YAML language support.",
```

Comment:
Does this get truncated? Probably would put important info first.

`Install YAML extension for additional AWS features` (no period)

Reply:
This message shouldn't get truncated (unless the window is extremely small). I'm not sold that this needs a reword.

Reply:
Though brevity without loss of clarity is a net gain . And "important info first" is a good practice.